allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
This is utility class to help with line breaking for long words, like java class names with package. This line height change should be somewhere else.